### ISSUE TITLE
尊敬的作者大大你好，关于训练的问题有一些不明白

### ISSUE THREAD

**Issue by anavanab99** [TachibanaYoshino/AnimeGAN#55]
3. Do edge_smooth
eg. python edge_smooth.py --dataset Hayao --img_size 256

4. Calculate the three-channel(BGR) color difference
eg. python data_mean.py --dataset Hayao
1，这两项是训练必须的步骤吗？ 搜遍全网也没搞清楚这两个步骤的意义，可能是我太菜了。是否预测的时候也需要执行这两个程序？
2，是我尝试迁移预测一些黑暗场景和爆炸场景，得到十分糟糕的效果，是训练集缺失的原因吧？假设我去热血漫搞点爆炸镜头能否改善这个问题呢？
3，训练集图片都挺小的，能否自定义输入图片的形状，比如直接输入1080p？
感谢作者，期望能得到你的教导。

**Comment by TachibanaYoshino** [comment]
Hi,
If you want to directly use the pre-trained model for style transfer, please run test.py for picture transfer and video2anime.py for for video transfer directly.
If you want to train your own model, you can follow the steps. Do edge_smooth is a necessary step, which is used during training. The channel(BGR) color difference is not necessary, it is only the preprocessing of the data, I will delete this step.
Regarding the dark and explosive scenes, it may be related to the problem you mentioned.
If you have enough GPU memory space, you can use larger resolution images for training, that is, you can customize the shape of the input image according to your dataset.

**Comment by TachibanaYoshino** [comment]
Hi,
If you want to directly use the pre-trained model for style transfer, please run test.py for picture transfer and video2anime.py for for video transfer directly.
If you want to train your own model, you can follow the steps. Do edge_smooth is a necessary step, which is used during training. The channel(BGR) color difference is not necessary, it is only the preprocessing of the data, I will delete this step.
Regarding the dark and explosive scenes, it may be related to the problem you mentioned.
If you have enough GPU memory space, you can use larger resolution images for training, that is, you can customize the shape of the input image according to your dataset.

**Comment by anavanab99** [comment]
Thank you for your answer. Do edge_ Is the smooth step prediction also necessary？
I'm very sorry I asked questions in Chinese. I thought you were my Chinese compatriots. Thank you for your answer. If I train a better model, I will share it with you at the first time

**Comment by TachibanaYoshino** [comment]
In the inference phase, Do edge_smooth is not needed. Do edge_smooth is the preprocessing of the animation dataset, which is used for edge promotion loss during training.

**Comment by anavanab99** [comment]
Thank you very much. You are a kind and handsome man.

**Comment by 1054171582** [comment]
anavanab99 同学我可以加你的联系方式吗，因为我也再接触这个项目我有一些问题不太明白想和您请教 微信13051093600 希望看到加我一下谢谢

**Comment by aluds123** [comment]
@TachibanaYoshino , Hi, I have some questions want to ask you:
"since the real photos in the training set are all landscape photos, if you want to stylize the photos with people as the main body, you may as well add at least 3000 photos of people in the training set and retrain to obtain a new model."
=> 
1. Do the photos of people in the training set also need to do "edge_smooth"?
2. The 3000 photos of people should be put into "AnimeGAN-master\dataset\train_photo" folder, right?

Thank you.

**Comment by schrott24preise** [comment]
Amazing Article!! This is really informative and knowledgeable for us, so thank you for sharing this information  with us, it will really work. Love Your Blog.
[Alufelgen Schrottpreis](https://schrott24preise.de/alufelgen-schrottpreis/)

**Comment by sonichappyhour** [comment]
**[Sonic's Corn Dogs](https://sonic-happyhour.com/sonic-drive-ins-corn-dog-edition/)** offer the perfect blend of crispy and juicy, making them an irresistible treat. Their affordable price adds to the charm, delighting food lovers everywhere!

**Comment by pkzones** [comment]
Are you looking for reliable product comparisons? Check out **[PkZones](https://pkzones.com/)**.com for the best deals and expert reviews!

**Comment by Statesmenucom** [comment]
<a href=" https://statesmenu.com/">States Menu</a> gives you accurate menu prices from restaurants across the USA. Compare costs for fast food, casual dining, and local spots before you order. Never overpay for a meal again—check prices first with States Menu!

**Comment by texasroadhousemenu** [comment]
你说的“训练”具体是指什么？

是机器学习/深度学习模型的训练？

是健身或其他类型的训练？

#55 是在哪个平台或项目下的编号？

比如 GitHub issue、某个论坛帖编号、某个课程章节等。

你不明白的具体是哪些内容？

可以复制粘贴该条内容或描述你不理解的部分。

[Texas Roadhouse Menus
](https://texas-roadhousemenus.com/)
